src/pages/explore/dfi/index.page.tsx

```diff
@@ -0,0 +1,33 @@
+import { HeroBanner, HeroBannerBg } from "@components/commons/HeroBanner";
+import { SSRConfig, useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export default function ExploreDFI() {
+  const { t } = useTranslation("page-explore-dfi");
+  return (
+    <div>
+      <HeroBanner
```

Comment:
hero banner desc font size should be smaller, can you help to check? maybe there was regression